ProyectoFinal/src/visual/ListarEmpresas.java

```diff
@@ -134,7 +161,8 @@ public void mouseClicked(MouseEvent e) {
 			btnVerSolicitudes.addActionListener(new ActionListener() {
 				public void actionPerformed(ActionEvent e) {
 					if(selectedEmpresa != null) {
-						// Abrir ventana
+						ListadoSolicitudesEmpresa listadoSolicitudesEmpresa = new ListadoSolicitudesEmpresa(selectedEmpresa);
+						listadoSolicitudesEmpresa.setVisible(true);
 
 						// Para evitar errores
 						setButtonsState(false);
```

Comment:
```suggestion
						setButtonsState(false);
						selectedEmpresa = null;
```

Es bueno también reiniciar el objeto de `Empresa`.